hardhat.config.ts

```diff
@@ -2,31 +2,64 @@ import "@nomiclabs/hardhat-ethers";
 import "@nomiclabs/hardhat-web3";
 import "@typechain/hardhat";
 import "hardhat-gas-reporter";
-import "solidity-coverage";
 import "@nomiclabs/hardhat-etherscan";
+
+// deployment plugins
+import "hardhat-deploy";
+import "hardhat-deploy-ethers";
+
+import "solidity-coverage";
+
 import { HardhatUserConfig } from "hardhat/types/config";
 
+const PRIVATE_KEY = process.env.PRIVATE_KEY;
+const INFURA_KEY = process.env.INFURA_KEY;
+
 const config: HardhatUserConfig = {
   solidity: {
-    version: "0.5.11",
-    settings: {
-      optimizer: {
-        enabled: true,
-        runs: 200,
-      },
-    },
+    compilers: [
+      { version: "0.5.11", 
```

Comment:
Bracket indentation fix needed.

Reply:
Should probably add this to the eslint config !